<ui version="4.0" >
 <class>Programmateur</class>
 <widget class="QMainWindow" name="Programmateur" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QPushButton" name="quitButton" >
    <property name="geometry" >
     <rect>
      <x>860</x>
      <y>590</y>
      <width>140</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text" >
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton" >
    <property name="geometry" >
     <rect>
      <x>860</x>
      <y>0</y>
      <width>140</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text" >
     <string>Open Hexfile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="programButton" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>860</x>
      <y>300</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text" >
     <string>Program</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readprogramButton" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>860</x>
      <y>340</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text" >
     <string>Read Program</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>20</y>
      <width>651</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label1" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>640</y>
      <width>671</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Please open a file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkHex" >
    <property name="geometry" >
     <rect>
      <x>860</x>
      <y>70</y>
      <width>140</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text" >
     <string>Check Hex File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton" >
    <property name="geometry" >
     <rect>
      <x>860</x>
      <y>180</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text" >
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eraseButton" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>860</x>
      <y>220</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text" >
     <string>Erase</string>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>860</x>
      <y>150</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Device Actions :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deviceIdButton" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>860</x>
      <y>260</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text" >
     <string>Device Id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dataButton" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>860</x>
      <y>380</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text" >
     <string>Write Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rDataButton" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>860</x>
      <y>420</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text" >
     <string>Read Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="configButton" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>860</x>
      <y>460</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text" >
     <string>Write Config</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rConfigButton" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>860</x>
      <y>500</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text" >
     <string>Read Config</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeCombo" >
    <property name="geometry" >
     <rect>
      <x>860</x>
      <y>540</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>PIC18F2550</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>PIC18F452</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>PIC16F876A</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="actionQuit" />
   </widget>
   <addaction name="menuFile" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen" >
   <property name="text" >
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
